strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/main/antlr4/org/carlspring/strongbox/nuget/filter/NugetODataFilter.g4

```diff
@@ -63,8 +63,8 @@ TAG
 
 ATTRIBUTE
 :
-    'Id'
-    | 'Version'
+    'id'
+    | 'version'
```

Comment:
@grantfar can you please clearify why this was changed?

Reply:
The database is using lowercase and if you have Id and Version in the query string instead of id and version it doesn't return the data you want